src/components/nav-bar/nav-bar.styles.scss

```diff
@@ -0,0 +1,26 @@
+.nav-bar-container {
+    width: auto;
+    height: auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-left: 8vw;
+}
+
+.logo {
+    background: url(../../assets/logo.svg);
+    width: 112px;
+    height: 112px;
+}
```

Comment:
this wont be necessary if you use an img tag

Reply:
deleted